sphericart-torch/include/sphericart/torch.hpp

```diff
@@ -18,7 +18,7 @@ class CudaSharedMemorySettings {
         : scalar_size_(0), l_max_(-1), grid_dim_x_(-1), grid_dim_y_(-1),
           requires_grad_(false), requires_hessian_(false) {}
 
-    bool update_if_required(torch::ScalarType scalar_type, int64_t l_max,
+    bool update_if_required(size_t scalar_size, int64_t l_max,
```

Comment:
this function needs documentation

Reply:
Added documentation in latest commit https://github.com/lab-cosmo/sphericart/commit/9da4642c88c808b6b5e38478e7899f269bc49f03.